index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Trace-Q</title>
```

Comment:
nit: TRACE-Q er med stort :nerd_face: 
```suggestion
        <title>TRACE-Q</title>
```